src/components/Card/CardsList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import CardWrapper from '../UI/CardWrapper';
+
+interface CardsListProps {
+  cards: {
+    firstPage: string;
+    secondPage: string;
+  }[];
+}
+
+const CardsList: React.FC<CardsListProps> = (props) => {
+  return (
+    <ul>
+      {props.cards.map((card: { firstPage: string; secondPage: string }) => (
```

Comment:
**nitpick:** Instead of passing props here, you can use props.children to enable better composition, and reduce prop drilling:

```tsx
const CardsList = (props: PropsWithChildren<CardsListProps>) => {
  return (
    <ul>
         {props.children}
    </ul>
)
```

Reply:
Not sure if I understand it. Should I map through  cards object in app component?

Reply:
> Not sure if I understand it. Should I map through cards object in app component?

You can render it higher (eg. in App.tsx), and reduce one level of prop drilling.

```tsx

<CardList>
 {props.cards.map((card: { firstPage: string; secondPage: string }) => <Card firstPage={firstPage} secondPage={secondPage}  />
</CardList>
```